OGame UI++/src/tab-stats.js

```diff
@@ -35,7 +36,7 @@ var fn = function () {
       var rentabilityTimes = [];
       for (var coords in config.my.planets) {
         var planet = config.my.planets[coords];
-        if (!planet.resources) {
```

Comment:
It's a bigger problem with multiple users signing on the same browser in. The effect is that the list of planets got merged and now I dont know why but planet.coords (for not my planet) is null and tab with stats doesn't work at all. I think config have to be saved per user.